scripts/config/msu/text.nut

```diff
@@ -13,4 +13,23 @@
 	{
 		return this.color(::Const.UI.Color.NegativeValue, _string);
 	}
+
+	function colorizeValue( _value, _invert = false, _addSign = true )
+	{
+		if (_value < 0) return this[_invert ? "colorGreen" : "colorRed"](_value);
+
+		if (_value > 0) return (_addSign ? "+" : "") + this[_invert ? "colorRed" : "colorGreen"](_value);
+
+		if (_value == 0) return this[_invert ? "colorRed" : "colorGreen"](_value);
+	}
```

Comment:
We should not be referring to functions using strings like this, no matter how convenient it is, it causes issues further down the line with it being unclear when this stuff is used as a function when searching.

Reply:
Ok, I will change it.